src/index.ts

```diff
@@ -14,5 +14,7 @@ const generate = (
   return generateSpec(workspace, options, projectName);
 };
 
+export * from './types/generator';
+export { Options };
```

Comment:
maybe export the index in the folder types and not just Options?

Reply:
You wanna export any types?